src/utils/appLinking.js

```diff
@@ -0,0 +1,125 @@
+import { Linking } from "react-native";
+import { Alert } from "../components";
+
+/**
+ * @description Object that holds the slack team id and several channel ids.
+ * 				The information in here is the most current as of May 28th, 2020.
+ * 				To find the team id or channel id login in slack on a computer.
+ * @typedef {{team: String, channel: String, fbGroupshpeucf: String}} slackInfo
+ * @const {slackInfo} slackInfo
+ */
+
+export const slackInfo = {
+	team: "TC61JSPUZ",
+	channel: {
+		general: "CPRDXHHD4",
+		announcements: "CC5S86GHE",
+		fundraisingCommitte: "CCUUWRU1Y",
+		motorshpe: "CP5HC76TD",
+		projects: "CNATZRD9C",
+		shpejr: "GCWFE5630"
+	},
+	fbGroupshpeucf: "120691161371846"
+};
+
+/**
+ * @description Function that returns the URI scheme and universal link for opening
+ *              an app, along with parameters, if included. Should not be changed
+ *              unless the apps change the universal links and/or URI schemas.
+ * @param {String}  inquiry  The value of the query.
+ * @param {String=} intent   The value that specify the type of query.
+ *
+ * @example Call the function like this formatUrl[name of the app](inquiry, intent)
```

Comment:
Remove "Call the function like this" and instead provide only the example of the call. If additional description is needed, use comments `//`